Mackbellemore/soen-390-team07
# feat(client): email notifications

#

## Changes this PR introduces
- Removed ShippingStore
- Send email when user place new sales, shipments
- Send email when user place defects that are 'Broken' only
- Send email when shipment status change to 'Delayed' or 'Cancelled' only




## How to test this PR

- You can test it using Netlify link
- You need to use a valid email address to verify if the email had been sent.



## ClickUp reference

CU-q53eaa

## Notes for reviewers

- Click up [link](https://app.clickup.com/t/q53eaa)



## Checklist
